src/app.types.tsx

```diff
@@ -235,6 +235,7 @@ export interface Session {
   summary: string;
   auto_saved: boolean;
   session_data: ImportSessionType;
+  _id?: string;
```

Comment:
Is `_id` really optional? I assume it's always returned - perhaps we should make it non-optional to ensure we're testing right & if we need to use an `_id` we don't have to null check

Reply:
The _id is not needed at all. The Session interface is used for the post request which initially does not have an ID anyways